client/src/components/AuthProvider.ts

```diff
@@ -16,7 +16,7 @@ import { getTokens, refreshToken } from "../connection/rest/auth";
 import { getCurrentUser } from "../connection/rest/identities";
 import { ConnectionType } from "../components/profile";
 
-const SECRET_KEY = "SASAuth";
+const SECRET_KEY = "SASAuth-1";
```

Comment:
Is this still needed based on the workaround that was identified?

Reply:
Nope. Since AuthProvider isn't out the wild anyway, we've effectively never used this key for customers. In future iterations, if we change the format of our cache, I'd recommend changing it...but for now, should be okay